testing/dart/canvas_test.dart

```diff
@@ -14,6 +14,8 @@ import 'package:vector_math/vector_math_64.dart';
 
 typedef CanvasCallback = void Function(Canvas canvas);
 
+bool get impellerEnabled => Platform.executableArguments.contains('--enable-impeller');
```

Comment:
Consider factoring this definition out into a `.dart` file under `testing/dart`, and importing it where needed.